content/20.results.md

```diff
@@ -59,10 +59,13 @@ We found that white honorees have been significantly overrepresented and Asian h
 
 ### Predicting Nationality with LSTM Neural Networks and Wikipedia
 
-We achieved a high overall accuracy on this test data with similar precision-recall values over the classes as we saw for our Wiki2019 dataset’s test set, giving confidence in our model.
+We achieved a high overall accuracy on this test data with similar precision-recall values over the classes as we saw for our Wiki2019 dataset's test set, giving confidence in our model.
```

Comment:
We should mention the average AUROC rather than accuracy. I think @arielah will output this value. Can do in subsequent PR.

Reply:
I haven't modified this section, but feel free to merge if a separate PR makes more sense.